test/release/examples/primers/modules.chpl

```diff
@@ -264,8 +264,8 @@ module MainModule {
       // writeln(_.bar); // also an error, _ is not an identifier otherwise
     }
 
-    /* The symbols provided by a ``use`` statement are only considered when
-       the name in question cannot be resolved directly within the local
+    /* The symbols provided by a private ``use`` statement are only considered
+       when the name in question cannot be resolved directly within the local
```

Comment:
Is this characterization necessarily true for functions with multiple overloads?  E.g., if the local scope contains a completely generic `foo()` that could be used to resolve a call, but a private use brings in a more specific `foo()`, the private use's would win, right?  So maybe this is saying something more strictly/simply than reality?

FWIW, if it helps (with this comment, or primer in general), I don't think the modules primer needs to teach everything about use/import x private/public.  If it tells enough to get users going in most cases and leaves them to figure out the corner cases themselves or by reading the spec, I think that's completely appropriate.  Or we could have a secondary/advanced primer (useVsImport.chpl?) to get into more detail.  Note that I haven't read this primer (recently), so may be anticipating a complication that doesn't exist.

Reply:
Good point and I can try to improve the comments.